docs/pages/database-access/guides/aws-opensearch.mdx

```diff
@@ -203,42 +203,36 @@ your terminal, and manually adjust `/etc/teleport.yaml`.
 
 Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
 
+<Tabs>
+  <TabItem label="Self-Hosted Edition" scope={["oss","enterprise"]}>
 ```code
-$ teleport db configure create \
+$ sudo teleport db configure create \
    -o file \
    --token=/tmp/token \
-   --proxy=<Var name="proxy-address" /> \
+   --proxy=<Var name="teleport.example.com" />:443 \
    --name=example-opensearch \
    --protocol=opensearch \
    --uri=your-opensearch-domain-url.eu-central-1.es.amazonaws.com:443 \
    --aws-account-id=(=aws.aws_access_key=)  \
    --labels=env=dev
 ```
-
-<Tabs>
-<TabItem label="Package Manager">
-
-On the host where you will run {{ service }}, start Teleport:
-
-```code
-$ sudo systemctl enable teleport
-$ sudo systemctl start teleport
-```
-
-</TabItem>
-<TabItem label="TAR Archive">
-
-On the host where you will run {{ service }}, create a systemd service
-configuration for Teleport, enable the Teleport service, and start Teleport:
-
+  </TabItem>
+  <TabItem label="Cloud"  scope={["cloud"]}>
 ```code
-$ sudo teleport install systemd -o /etc/systemd/system/teleport.service
-$ sudo systemctl enable teleport
-$ sudo systemctl start teleport
+$ sudo teleport db configure create \
+   -o file \
+   --token=/tmp/token \
+   --proxy=<Var name="mytenant.teleport.sh" />:443 \
+   --name=example-opensearch \
+   --protocol=opensearch \
+   --uri=your-opensearch-domain-url.eu-central-1.es.amazonaws.com:443 \
+   --aws-account-id=(=aws.aws_access_key=)  \
+   --labels=env=dev
 ```
+  </TabItem>
+  </Tabs>
```

Comment:
I think we can replace this whole `Tabs` with the content of the `TabItem` for "Cloud", using `example.teleport.sh` for the variable value. Self-hosting readers should understand that this is a domain name, and Cloud users will see the expected format. (Using `example` instead of `mytenant` because Xin wants to move away from the "tenant" language.)

I would add explicit guidance to assign `--proxy` to the domain name of your Teleport cluster.